### ISSUE TITLE
(UNPLAYABLE MAC OS) Switch to desktop everytime the cursor touch top(menubar), makes game unplayable

### ISSUE THREAD

**Issue by VelaYF** [ValveSoftware/Dota2-Gameplay#11122]
### Description

When the mouse cursor touches the top of the screen(the area of the menu bar), the game will be shanked in the dock.

Device: Macbook Pro 2021 14`
OS Version: Mac os 13.5.1

**Comment by VelaYF** [comment]
It looks like you guys changed the full-screen display mode, now the game can be hidden while running!
Great job!
Hope you can solve this issue ASAP!

**Comment by alindmtr** [comment]
Yes, exclusive full screen does not behave properly anymore. It's more like a borderless windows mode

**Comment by DamirLutfullin** [comment]
same problem

**Comment by alindmtr** [comment]
Still not working

**Comment by DamirLutfullin** [comment]
there is no decision?

**Comment by ozodrukh** [comment]
we should get more attention and thumb up reactions to get it fixed i believe.

In addition to described bug, sometimes cursor changes from Dota to system small icon, and it's too small to notice in game

**Comment by alindmtr** [comment]
bump

**Comment by sonnyblackbonnes** [comment]
Unplayable

**Comment by lqrhy3** [comment]
Stuppidest game devs in the world
